### PR TITLE
e2e/testing: A few small patches

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by jlebon** [comment]
No new features, simply a docs change, typo fix, and CI debugging aid.
/lgtm

File: pkg/apis/machineconfiguration.openshift.io/v1/types.go [hunk 1]
```diff
@@ -304,8 +304,9 @@ type MachineConfigPoolSpec struct {
 	// Label selector for Machines.
 	MachineSelector *metav1.LabelSelector `json:"machineSelector,omitempty"`
 
-	// If true, changes to this machine pool should be stopped.
-	// This includes generating new desiredMachineConfig and update of machines.
+	// If true, changes to this machine pool should be stopped,
+	// including generating new desiredMachineConfig and update of machines.
+	// This flag is intended for administrators to change.
```
Comment (abhinavdahiya) [inline pkg/apis/machineconfiguration.openshift.io/v1/types.go]: wait, IMO all the fields in spec are for edit. not just this...

Comment (cgwalters) [reply pkg/apis/machineconfiguration.openshift.io/v1/types.go]: Hm...I thought I saw something else in the operator changing the spec, but trying now I can't find anything.

Dropped that patch out and updated :arrow_down:

**Comment by cgwalters** [comment]
> @cgwalters could we put something about testing/e2e/etc in the PR title so it's easier to find in github list later?

ashcrow did that, ok now?

**Comment by kikisdeliveryservice** [comment]
@cgwalters great! :+1:

**Pull request merged**